src/main/resources/db/changelog/db.changelog-master.yaml

```
@@ -0,0 +1,3 @@
+databaseChangeLog:
+  - include:
+      file: db/changelog/v.0.1/db.changelog-master-v.0.1.yaml
```

Path indicates what version is used, postfix is excessive.

```suggestion
      file: db/changelog/v.0.1/db.changelog-master.yaml
```

---

done